### PR TITLE
fix redundancy boot proposal if there is more devices (bnc#917025)

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: package/yast2-bootloader.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Feb 12 12:45:50 UTC 2015 - jreidinger@suse.com
+
+- fix redundancy boot proposal if there is more devices
```
Comment (lslezak) [inline package/yast2-bootloader.changes]: Bad English: ... **are** more devices.

Comment (jreidinger) [inline package/yast2-bootloader.changes]: right, thanks for notice

**Comment by lslezak** [comment]
LGTM

**Comment by jreidinger** [comment]
thanks

**Pull request merged**